tests/datasets-validate-postmatch/test.rules

```diff
@@ -0,0 +1 @@
+alert http any any -> $HOME_NET any (msg:"HTTP learning"; flow:established,to_client; http.content_type; content:"noone"; http.server; content:"ECS"; fast_pattern; dataset:set,http,type string,state output/http.intel; sid:2; rev:1; priority:2;)
```

Comment:
We should keep the filename to `http.intel` only as `output` dir doesn't exist initially and is written everytime we run the test.